Dockerfile

```diff
@@ -24,12 +25,19 @@ RUN export GOOS=$(echo ${TARGETPLATFORM} | cut -d / -f1) && \
 
 WORKDIR /go/src/github.com/viaduct-ai/kustomize-sops
 
-ADD . .
+COPY go.mod go.sum ./
+RUN go mod download
 
-# Perform the build
+COPY . .
```

Comment:
What's the purpose of `COPY`ing the go.mod go.sum over twice?

Wouldn't this suffice

```Dockerfile
COPY . .
RUN go mod download
# rest
```

Reply:
I was initially thinking we could have the dependencies in an early layer and copy the code after, but I think it's probably reasonable to just copy everything and do a go mod download. I'll push a fix now.